common/thorhelper/thorread.hpp

```diff
@@ -27,19 +27,56 @@
 #include "jrowstream.hpp"
 #include "rtlkey.hpp"
 
+//The following is constant for the life of a disk read activity
+interface IDiskReadOutputMapping : public IInterface
```

Comment:
style question: I use 'extends' in this context. We should be consistent, what what is preferred?

Reply:
I prefer to avoid non-standard things like extends - they just make the code less accessible for new readers.

Reply:
yes, extends would be better

Reply:
Should we remove the use of extends from within our code?

Reply:
'interface' is non-standard too.. I feel it (and personally 'extends') clarifies the intended pattern.